makefile

```diff
@@ -57,6 +57,7 @@ gen-api:
 	oapi-codegen --config codegen/configs/vcr_v2.yaml docs/_static/vcr/v2.yaml | gofmt > vcr/api/v2/generated.go
 	oapi-codegen --config codegen/configs/auth_v1.yaml docs/_static/auth/v1.yaml | gofmt > auth/api/v1/generated.go
 	oapi-codegen --config codegen/configs/didman_v1.yaml docs/_static/didman/v1.yaml | gofmt > didman/api/v1/generated.go
+	oapi-codegen --config codegen/configs/crypto_store_client.yaml docs/_static/crypto/nuts-storage-api-v1.yaml | gofmt > crypto/storage/httpclient/generated.go
```

Comment:
Why the `httpclient` package isntead of putting it just into `storage`?

Reply:
All generated clients are called `generated.go`, these files also get ignored during code scannen etc. I wanted to follow this convention. But than you get `/crypto/storage/generated.go` and that seemed weird. So I gave it its own package to live in.

Reply:
Decided: move it to a better name: `external`